### DISCUSSION TITLE
Rancher Desktop Not Starting Post update to 1.9.1

### DISCUSSION THREAD

**Discussion by GuArDiAn1611** [rancher-sandbox/rancher-desktop#5481] [Q&A]
The rancher was working fine before updating to 1.9.1, Facing the below issue

<img width="761" alt="Screenshot 2023-09-07 at 12 59 05 PM" src="https://github.com/rancher-sandbox/rancher-desktop/assets/59787286/739bf173-3b85-48a0-a312-ac1027708d9d">

**Error Starting Kubernetes**
Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl.ventura exited with code 1

**Last command run:**
/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl.ventura start --tty=false 0

Context:
Starting virtual machine

Some recent [logfile](app://./index.html#) lines:
```
    '--tty=false',
    '0'
  ],
  stdout: '',
  stderr: 'time="2023-09-07T12:43:30+05:30" level=info msg="Using the existing instance \\"0\\""\n' +
    'time="2023-09-07T12:43:30+05:30" level=fatal msg="errors inspecting instance: [host agent is running but driver is not]"\n',
  code: 1,
  [Symbol(child-process.command)]: '/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl.ventura start --tty=false 0'
}
```

Please let me know if you need anything else to fix this issue.

**Comment by AntsKnows** [comment]
any solutions?  i've met the same problem

**Comment by jomo02** [comment]
Same problem for me (using Rancher Desktop 1.11.1 on a M3).

**Comment by pjiricka** [comment]
Same here, Rancher Desktop 1.12.1 on an Intel Mac, macOS 14.2.1.

**Reply by pjiricka** [reply]
Good news, I was able to work around this by switching to VZ virtualization and socket-vmnet networking.

**Reply by mttrhll** [reply]
Thx pjiricka! ... I was also able to get Rancher Desktop 1.12.3 running on an M3 silicon with macOS 14.3.1 using the settings you mention. Posting this for others in case they get stuck. 

My process was:

1. Completely uninstall previous versions (as best as I could)
2. Backed up my .kube/ just in case
3. Install Rancher Desktop 1.12.3 using all the defaults in the installation process
4. It will fail to start properly
5. Go to "Troubleshooting" and do a "Factory Reset" (just in case)
6. Go to "Preferences" -> "Virtual Machine" and make the changes you describe in the "Emulation" and "Network" sections.
7. Quit RD and then start it again... it will take a little bit 

It seems to be working pretty well. Thx again ur combo of settings is what I guess is needed... I wasn't sure because of VZ being marked as experimental. 

I also set "Enable Rosetta support" ... not sure if that matters.

**Reply by alochaus** [reply]
It only works for me if rosetta support is enabled.
